assistive_gym/envs/human_comfort_env.py

```diff
@@ -70,6 +74,86 @@ def _get_obs(self, agent=None): # not needed
     def get_total_force(self):
         pass
 
+    
+    def get_reba_score(self):
+        human_dict = HumanUrdfDict()
+        rebaScore = RebaScore()
+        # list joints in the order required for a reba score
+        joints = ["head", "neck", "left_shoulder", "left_elbow", "left_lowarm", "right_shoulder", "right_elbow", "right_lowarm", 
+            "left_hip", "left_knee", "left_ankle", "right_hip", "right_knee", "right_ankle", "left_hand", "right_hand"]
+        
+        # obtain the links in the right order for the rebascore code
+        dammy_ids = []
+        for joint in joints:
+            dammy_ids.append(human_dict.get_dammy_joint_id(joint))
+
+        # use dammy ids to obtain the right link, use the right knee as the root joint
+        pose = []
+        root = p.getLinkState(self.human.body, dammy_ids[12])[0] # root joint
+
+        for i in dammy_ids:
+            # get the location of each dammy joint and append to the pose list
+            loc = p.getLinkState(self.human.body, i)[0]
+            norm_loc = [loc[0] - root[0], loc[1] - root[1], loc[2] - root[2]]
```

Comment:
Is that the relative location to the root that you are trying to get? 
p.getLinkState()[0] will return you the center of mass location (mid point of the limb) I suspect. I'm not sure if you need to get the end point or mid point is fine
I think we might need to check again to be sure.
Here is the doc 
https://docs.google.com/document/d/10sXEhzFRSnvFcl3XxNGhnD4N2SedqwdAvK3dsihxVUA/edit?pli=1#heading=h.3v8gjd1epcrt